tests/unit/synapseclient/extensions/conftest.py

```diff
@@ -0,0 +1,92 @@
+import os
+import sys
+from unittest.mock import Mock
+
+import pytest
+
+# from synapseclient.extensions.curator.schema_generation import load_df
+from synapseclient.extensions.curator.schema_generation import (
+    DataModelGraph,
+    DataModelGraphExplorer,
+    DataModelParser,
+    load_df,
+)
+
+TESTS_DIR = os.path.dirname(os.path.abspath(__file__))
+SCHEMA_FILES_DIR = os.path.join(TESTS_DIR, "schema_files")
```

Comment:
There's no more `data` directory. I organized everything under the new `schema_files` directory